src/Microsoft.AspNet.Hosting.Abstractions/IApplicationLifetime.cs

```diff
@@ -24,5 +25,16 @@ public interface IApplicationLifetime
         /// </summary>
         /// <returns></returns>
         CancellationToken ApplicationStopped { get; }
+
+        /// <summary>
+        /// Sends the notification that the application has started
```

Comment:
I know the code is still being iterated on, but remember that doc comments need to be full sentences, and thus end with periods